dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -54,3 +57,13 @@
                     %span.puzzle-number
                       = @script_level.position
                 %td.name= link_to student.name, build_script_level_path(@script_level, user_id: student.id, section_id: @section.id)
+  :javascript
+    function resizeScrollable() {
+      var newHeight = $('.teacher-panel').height() -
+          $('.teacher-panel h3').outerHeight() -
+          25 - /* magic... */
```

Comment:
Hahah, any best guess of what this might grow/shrink? Maybe marginBelowScrollablePanel?